File: C#/08-Laço While/Contador.cs
using System;

class Contador
{
    static void Main(string[] args)
    {
//Podemos fazer repetições em C# através do laço while
//Vamos fazer um contador de 0 a 10
        int cont = 0;
//Enquanto variavel cont for menor ou igual a 10, cont recebe 1
        while(cont <= 10)
        {
            Console.WriteLine(cont +" ");
            cont++;
        }  
        Console.ReadLine();      
    }
}

File: Java/08-Laço While/Contador.java
class Contador {
    public static void main(String[] args) {
        // Vamos usar o laço while para fazer uma contagem até 10
        int contador = 0;

        // Enquanto o contador for menor ou igual 10 ele vai incrementar
        while (contador <= 10) {
            System.out.println(contador);
            contador++;
        }
        // Quando atingir valor especificado(10) o laço se encerra

        System.out.println("Contagem finalizada!");
    }
}